build_projects/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -152,12 +152,10 @@ public static BuildTargetResult DownloadHostAndSharedFxArchives(BuildTargetConte
 
             Mkdirp(Path.GetDirectoryName(combinedSharedHostAndFrameworkArchiveFile));
 
-            AzurePublisher.DownloadFile(
-                AzurePublisher.CalculateArchiveBlob(
-                    combinedSharedHostAndFrameworkArchiveFile, 
-                    sharedFrameworkChannel, 
-                    sharedFrameworkVersion), 
-                combinedSharedHostAndFrameworkArchiveFile).Wait();
+            new AzurePublisher().DownloadFile(combinedSharedHostAndFrameworkArchiveFile, 
```

Comment:
Use the static method here, an instance of AzurePublisher depends on some environment variables which are not present in Jenkins.